backend/globaleaks/handlers/base.py

```diff
@@ -188,7 +189,8 @@ def validate_type(value, type):
         elif callable(type):
             retval = BaseHandler.validate_python_type(value, type)
             if not retval:
-                log.err("-- Invalid python_type, in [%s] expected %s", value, type)
+                log.err(
+                    "-- Invalid python_type, in [%s] expected %s", value, type)
```

Comment:
@joeman65 @alessiofranceschini: This seems to be just stilistic  code review, please provide this separately to not misk code review of our code with the features you need to implement.